contribution.md

```diff
@@ -0,0 +1,6 @@
+## Project Contributions
```

Comment:
rename to `projects/python/aither.md` or whatever language it is in. Also provide a link.